pkg/gui/services/custom_commands/client.go

```diff
@@ -47,3 +64,45 @@ func (self *Client) GetCustomCommandKeybindings() ([]*types.Binding, error) {
 
 	return bindings, nil
 }
+
+func (self *Client) showCustomCommandsMenu(customCommandsMenu config.CustomCommandsMenu) error {
+	menuItems := make([]*types.MenuItem, 0, len(customCommandsMenu.Commands))
+	for _, command := range customCommandsMenu.Commands {
+		if command.Context != "" && command.Context != "global" {
+			viewNames, err := self.keybindingCreator.getViewNamesAndContexts(command)
+			if err != nil {
+				return err
+			}
+
+			currentView := self.c.GocuiGui().CurrentView()
+			enabled := currentView != nil && lo.Contains(viewNames, currentView.Name())
+			if !enabled {
+				continue
+			}
```

Comment:
Here we omit commands that are not applicable in the current context. At first I was a bit torn about this, and considered including them with a DisabledReason so that they appear crossed out. However, omitting them is similar to what we do in the global keybindings menu, and actually I do find it important, because it allows us to configure different commands with the same key in different contexts.